llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -7827,8 +7759,9 @@ DenseMap<const SCEV *, Value *> LoopVectorizationPlanner::executePlan(
   //===------------------------------------------------===//
 
   // 2. Copy and widen instructions from the old loop into the new loop.
-  BestVPlan.prepareToExecute(ILV.getTripCount(),
-                             ILV.getOrCreateVectorTripCount(nullptr), State);
+  BestVPlan.prepareToExecute(
+      ILV.getTripCount(),
+      ILV.getOrCreateVectorTripCount(ILV.LoopVectorPreHeader), State);
```

Comment:
How/Is this change related?

Reply:
Previously getOrCreateVectorTripCount would be called by createInductionResumeVPValues, which has now been removed, so we need to pass `ILV.LoopVectorPreHeader` here create the vector trip count IR

Reply:
Ahh, previously vector trip count was created earlier, so here it was effectively only getting it, so null could be passed.

Can commit this independently, removing the dependence that this not be the first time it is called.

Reply:
Done in f9120dc2a60aedcab5ce99e40b6a2bd3849f0bb9, thanks!